packages/keystone/plugins/utils/address-service-client/AddressServiceClient.js

```diff
@@ -102,9 +96,9 @@ class AddressServiceClient {
             throw new Error('The `s` parameter is mandatory')
         }
 
-        const urlParams = [`s=${s}`, this.urlifyParams(params)].filter(Boolean)
+        const urlParams = new URLSearchParams({ s, ...params }).toString()
```

Comment:
URLSearchParams - do not work recursive
<img width="483" alt="image" src="https://user-images.githubusercontent.com/1640424/206629617-0cf38aef-abad-42b2-a10c-dd1c6317ebed.png">

Reply:
🤦‍♂️
will fix soon

Reply:
Recursivity is not needed here, because we pass a one-level object with strings (according to jsdoc).